rudder-web/src/main/scala/bootstrap/liftweb/checks/ResumePolicyUpdateRunning.scala

```diff
@@ -0,0 +1,81 @@
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* In accordance with the terms of section 7 (7. Additional Terms.) of
+* the GNU General Public License version 3, the copyright holders add
+* the following Additional permissions:
+* Notwithstanding to the terms of section 5 (5. Conveying Modified Source
+* Versions) and 6 (6. Conveying Non-Source Forms.) of the GNU General
+* Public License version 3, when you create a Related Module, this
+* Related Module is not considered as a part of the work and may be
+* distributed under the license agreement of your choice.
+* A "Related Module" means a set of sources files including their
+* documentation that, without modification of the Source Code, enables
+* supplementary functions or services in addition to those offered by
+* the Software.
+*
+* Rudder is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
+
+*
+*************************************************************************************
+*/
+
+package bootstrap.liftweb
+package checks
+
+import net.liftweb.common._
+import java.io.File
+import com.normation.cfclerk.services.UpdateTechniqueLibrary
+import com.normation.rudder.batch.AsyncDeploymentAgent
+import javax.servlet.UnavailableException
+import com.normation.eventlog.ModificationId
+import com.normation.utils.StringUuidGenerator
+import com.normation.rudder.domain.eventlog.RudderEventActor
+import com.normation.rudder.batch.AutomaticStartDeployment
+
+/**
+ * Check at webapp startup if Rudder Technique library has to be reloaded
+ * If flag file is present then reload technique library
+ * This needs to be achieved after all tasks that could modify configuration (ie: CheckMigrationDirectiveInterpolatedVariablesHaveRudderNamespace)
+ */
+class ResumePolicyUpdateRunning(
+    asyncGeneration : AsyncDeploymentAgent
+  , uuidGen         : StringUuidGenerator
+) extends BootstrapChecks with Loggable {
+
+  override def checks() : Unit = {
+
+    val filePath = asyncGeneration.policyUpdateRunningFlagPath
+    // Check if the force technique reload file is present, and reload technique library if needed
+    val file =  new File(filePath)
+    try {
+      if (file.exists) {
+        // File exists, reload
+        logger.info(s"Flag file '${filePath}' found, Start a new policy update now")
+        asyncGeneration ! AutomaticStartDeployment(ModificationId(uuidGen.newUuid), RudderEventActor)
+      } else {
+        logger.info(s"Flag file '${filePath}' does not exists, No need to start a new policy update")
```

Comment:
Not sure we want that at info level. Is it the same level for the equivalent message for technique library reload ?